app/views/requests/_new.html.erb

```diff
@@ -0,0 +1,18 @@
+<div id="newRequest" class="modal fade">
+  <div class="modal-dialog modal-dialog-scrollable">
+    <div class="modal-content">
+      <div class="modal-header">
+        <h4 class="modal-title">New Quantity Request</h4>
+        <button type="button" class="close" data-bs-dismiss="modal" aria-label="Close">
+          <span aria-hidden="true">&times;</span>
+        </button>
+      </div>
+      <div class="modal-body p-4">
+        <%= form_with(url: requests_path) do |form| %>
+          <%= form.collection_select(:partner_id, @partners, :id, :name, {include_blank: "Select a partner", required: true}, {class: "form-control mb-2"} ) %>
```

Comment:
Functional requirement that was missed in the issue:  This should be the *active* partners only.